website/views/options/social/index.jade

```diff
@@ -39,7 +39,7 @@ script(type='text/ng-template', id='partials/options.social.party.html')
       p
         =env.t('startLFG')
         |&nbsp;
-        a(href="http://habitrpg.wikia.com/wiki/Board:The_Archery_-_LFG")=env.t('linkLFG')
+        a(href="https://habitrpg.com/#/options/groups/guilds/f2db2a7f-13c5-454d-b3ee-ea1f5089e601")=env.t('linkLFG')
```

Comment:
The text for `linkLFG` is: 
> To advertise your new party or find one to join, go to  The Archery  on the Wiki.

Since the link has been changed, the text needs updating too. You can find the lines that needs to be changed here: https://github.com/HabitRPG/habitrpg/blob/develop/common/locales/en/groups.json#L25-L27

@lemoness Do you have something specific you want the text to say?

@remmz0r Feel free to recommend some text, but I'd hold off on committing anything until it's approved by lemoness.